src/swarm/neo/protocol/socket/MessageSender.d

```diff
@@ -5,6 +5,18 @@
     This class is suitable for both half and full duplex because it does not
     handle epoll notifications directly (it is not a select client).
 
+    Once upon the time there was an explicit flush method. It relied on the
```

Comment:
I believe the right wording is "Once upon a time" :P

Reply:
I'm just happy I put any article at all ;-)

Reply:
Fixed, thanks!